cmd/ctr/commands/commands.go

```diff
@@ -178,7 +178,7 @@ func LabelArgs(labelStrings []string) map[string]string {
 
 // PrintAsJSON prints input in JSON format
 func PrintAsJSON(x interface{}) {
-	b, err := json.MarshalIndent(x, "", "    ")
```

Comment:
I would just leave it as is. This whole file is a copy if the corresponding file from containerd: https://github.com/containerd/containerd/blob/v1.5.0-beta.0/cmd/ctr/commands/commands.go#L206

Reply:
we can compress it when save or output OCI image tar? not affect input json